pages/series/[slug].vue

```diff
@@ -63,6 +71,86 @@ const parsedCarouselData = computed(() => {
     }
   })
 })
+
+// Transform Data for Tabbed Section
+const parsedUpcomingEvents = computed(() => {
+  // fail gracefully if data does not exist (server-side)
+  if (!upcomingEvents.value)
+    return []
+
+  // Transform data
+  return upcomingEvents.value.map((item, index) => {
+    return {
+      ...item,
+      image: item.image[0]
+    }
+  })
+})
+const parsedPastEvents = computed(() => {
+  // fail gracefully if data does not exist (server-side)
+  if (!pastEvents.value)
+    return []
+
+  // Transform data
+  return pastEvents.value.map((item, index) => {
+    return {
+      ...item,
+      image: item.image[0]
+    }
+  })
+})
+
+// Transform data for Other Series Section
+// This section only shows 3 items max, and prioritizes upcoming events over ongoing
+const parsedOtherSeries = computed(() => {
+  // fail gracefully if data does not exist (server-side)
+  if (!otherSeriesUpcoming.value && !otherSeriesOngoing.value)
+    return []
+
+  let otherSeries = otherSeriesUpcoming.value.concat(otherSeriesOngoing.value)
+  // Get first 3 events
```

Comment:
we need to further filter the array by removing current series like 
` // Return series without the page's featured event
  otherSeries = otherSeries.filter(({ to }) => !to.includes(route.params.slug) )`

Reply:
✅